### PR TITLE
ANA-2987: Client demo notebook of Mortgage Backed Securities

### PR DESCRIPTION
**Pull request by fabiodiasfinbourne**
# Pull Request Checklist

- [ ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [ ] Raised the PR against the `develop` branch
- [ ] Notebook outputs do not contain any priviledged data

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/use-cases/instruments/Mortgage Backed Securities.ipynb [hunk 1]
```diff
@@ -0,0 +1,1152 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>can you add the docstring with description as per the style guide as this generates the index page</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>this has the possibility to cause concurrency issues when builds run in concourse and GitHub at the same time (something we've observerd). put the portfolios codes into a variable with a random suffix and delete at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>"lusid" &gt; "LUSID"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    trade_date_1 = datetime.strptime("2023-05-28T00:00:00+00:00","%Y-%m-%dT%H:%M:%S%z")
</pre><p> </p><p>you can create utc dates using</p><pre class="ql-syntax" spellcheck="false">datetime(2023, 5, 28, tzinfo=timezone.utc)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>did you intend to show the output here (and below)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>did you mean to show the output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>show the output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>yes, shall I change it to</p><pre class="ql-syntax" spellcheck="false">display(lusid_response_portfolio_1[["transaction_id", "settlement_date", "units", "transaction_currency"]])
</pre><p>(and below)?</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>Again, shall I change it to</p><pre class="ql-syntax" spellcheck="false">display(get_val("2023-07-06T10:00:00Z", "portfolio_1"))
</pre><p>?</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>noted, will do</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>noted, will do</p>

Comment (rizwansaeed) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>sure, i think it was just a case of the cell not being run before you saved it (or it was cleared before you saved it)</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>Ah yeah, I intentionally cleared all outputs before I saved it. I'll rerun and not clear.</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>Will do</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>Done</p>

Comment (fabiodiasfinbourne) [reply examples/use-cases/instruments/Mortgage Backed Securities.ipynb:1]: <p>All done</p>

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**